File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarCamarasOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarCamarasOut : ParametroBaseOut
    {
        public List<Camara> Camaras { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarCamarasOut.ts
import { Camara } from "src/app/Components/Camara/class/Camara";
import { ParametroBaseOut } from "./ParametroBaseOut";

export class ListarCamarasOut extends ParametroBaseOut{

    private camaras: Camara[];

    constructor(){
        super();
        this.camaras = [];
    }

    public get Camaras(): Camara[] {
        return this.camaras;
    }
    public set Camaras(value: Camara[]) {
        this.camaras = value;
    }

}